TACC/Core-CMS
# Task/ecepweb-205-logos

## Overview

- Swap ECEP logo for `.svg` that also has words.
- Add ECEP favicon.[^1]

## Related

- [ECEPWEB-205](https://jira.tacc.utexas.edu/browse/ECEPWEB-205)
- required by https://github.com/TACC/Core-CMS-Resources/pull/149

## Changes

- update ECEP logo(s)

## Testing

1. Confirm [ECEP] **still** has white logo with no words.
2. Confirm [ECEP] logo is an `.svg`.
3. Confirm [ECEP] favicon is white "ECEP" on purple field.

## Screenshots

| Local | [Remote][ECEP] |
| - | - |
| ![ECEP Logos - Local](https://user-images.githubusercontent.com/62723358/176775230-d621dc5f-ae8e-4106-8a70-aec6cec2973a.png) | ![ECEP Logos - Remote](https://user-images.githubusercontent.com/62723358/176775842-6a50a54a-e3e6-46a7-9f13-6d488563eab3.png) |

## Notes

[Design review requested privately.](https://user-images.githubusercontent.com/62723358/176763980-054f07dd-4f3d-4ab7-85bf-4dfdb7025e5d.png) They opted for B "Use logo without text".

[ECEP]: https://pprd.ecep.tacc.utexas.edu/

[^1]: Minimal implementation, just an `.ico`, because Core-CMS is not setup for more options yet.
